stellargraph/layer/gcn_lstm.py

```diff
@@ -0,0 +1,347 @@
+        ``method='gcn'`` argument to do this pre-processing.
+
+      - The nodes provided to the :class:`FullBatchNodeGenerator.flow` method are
+        used by the final layer to select the predictions for those nodes in order.
+        However, the intermediate layers before the final layer order the nodes
+        in the same way as the adjacency matrix.
+
+    Args:
+        layer_sizes (list of int): Output sizes of GCN layers in the stack.
+        generator (FullBatchNodeGenerator): The generator instance.
+        bias (bool): If True, a bias vector is learnt for each layer in the GCN model.
+        dropout (float): Dropout rate applied to input features of each GCN layer.
+        activations (list of str or func): Activations applied to each layer's output;
+            defaults to ['relu', ..., 'relu'].
+        kernel_initializer (str or func, optional): The initialiser to use for the weights of each layer.
+        kernel_regularizer (str or func, optional): The regulariser to use for the weights of each layer.
+        kernel_constraint (str or func, optional): The constraint to use for the weights of each layer.
+        bias_initializer (str or func, optional): The initialiser to use for the bias of each layer.
+        bias_regularizer (str or func, optional): The regulariser to use for the bias of each layer.
+        bias_constraint (str or func, optional): The constraint to use for the bias of each layer.
+    """
+    
+    def __init__(
+        self,
+        outputs,
+        adj,
+        layer_sizes=[10,200],
+        bias=True,
+        dropout=0.5,
+        activations=["relu","relu","sigmoid"],
+        kernel_initializer=None,
+        kernel_regularizer=None,
+        kernel_constraint=None,
+        bias_initializer=None,
+        bias_regularizer=None,
+        bias_constraint=None,
+    ):
+
+        super(Graph_Convolution_LSTM, self).__init__()
+
+        self.layer_sizes = layer_sizes
+        self.activations = activations
+        self.bias = bias
+        self.dropout = dropout
+        self.outputs = outputs
+
+        self.kernel_initializer = initializers.get(kernel_initializer)
+        self.kernel_regularizer = regularizers.get(kernel_regularizer)
+        self.kernel_constraint = constraints.get(kernel_constraint)
+        self.bias_initializer = initializers.get(bias_initializer)
+        self.bias_regularizer = regularizers.get(bias_regularizer)
+        self.bias_constraint = constraints.get(bias_constraint)
+
+        self.gcn_1 = GraphConvolution(units = layer_sizes[0], A=adj, activation=self.activations[0])
+        self.gcn_2 = GraphConvolution(units = layer_sizes[0], A=adj, activation=self.activations[1])
+        self.lstm = LSTM(self.layer_sizes[1], return_sequences=False)
+        self.dense = Dense(self.outputs, activation=self.activations[2])
+        self.dropout = Dropout(self.dropout)
+
+    def call(self, inputs):
```

Comment:
Echoing the comment about the name, our "model" classes usually have two methods:

```python
def __call__(self, inputs): # like this `call`, but note the double-underscore, to make it work like `some_model(inputs)`
    ...

def build(self): # builds appropriate input tensors and then applies the model
    inputs = ...
    outputs = self(inputs)
    return inputs, outputs
```